packages/file/src/index.coffee.md

```diff
@@ -394,7 +394,7 @@ require('nikita')
               else
                 throw err
             null
-      utils.string.render.call @, config if config.context
+      config.content = config.transform.call undefined, config: config if config.transform
```

Comment:
ideally, we shall add 2 things, could be in another PR:

1. surround code with a try/catch
2. handle promises